exercises-latex/12-python/loesung.py

```diff
@@ -34,19 +34,21 @@ def f(t, a, b, c, d):
 
 t_plot = np.linspace(-0.5, 2 * np.pi + 0.5, 1000) * 1e-3
 
-plt.errorbar(
+fig, ax = plt.subplots(1, 1, constrained_layout=True)
```

Comment:
`constrained_layout=True` is already deprecated, it should be `layout="constrained"` nowadays